apps/report-documents-not-received/views/accessibility.html

```diff
@@ -0,0 +1,55 @@
+{{<layout}}
+
+  {{$header}}
+    {{header}}
+  {{/header}}
+
+  {{$content}}
+    <p class="govuk-body">{{#t}}accessibility.intro{{/t}}</p>
+    <ul class="govuk-list govuk-list--bullet">
+        <li>{{#t}}accessibility.intro-p2-li1{{/t}}</li>
+        <li>{{#t}}accessibility.intro-p2-li2{{/t}}</li>
+        <li>{{#t}}accessibility.intro-p2-li3{{/t}}</li>
+        <li>{{#t}}accessibility.intro-p2-li4{{/t}}</li>
+        <li>{{#t}}accessibility.intro-p2-li5{{/t}}</li>
+    </ul>
+    <p>{{#t}}accessibility.intro-p3-pre-link{{/t}}<a href="https://mcmw.abilitynet.org.uk/" target="_blank"> {{#t}}accessibility.intro-p3-link{{/t}}</a> {{#t}}accessibility.intro-p3-post-link{{/t}}</p>
```

Comment:
Applicable for all instances - `target="_blank"` Is this a requirement to open these links in a new tab?

Reply:
Not a requirement no, will change that.